src/LaunchDarkly.EventSource/ExponentialBackoffWithDecorrelation.cs

```diff
@@ -22,12 +22,18 @@ public ExponentialBackoffWithDecorrelation(TimeSpan minimumDelay, TimeSpan maxim
         /// </summary>
         public TimeSpan GetNextBackOff()
         {
-            int nextDelay = Convert.ToInt32(Math.Min(_maximumDelay.TotalMilliseconds, _minimumDelay.TotalMilliseconds * Math.Pow(2, _reconnectAttempts)));
```

Comment:
I broke out this calculation into an `internal` method so that the tests can use it.